src/datastore/postgres-store.ts

```diff
@@ -2842,14 +2896,15 @@ export class PgDataStore extends (EventEmitter as { new (): DataStoreEventEmitte
       status,
       canonical,
       index_block_hash,
+      atch_resolved,
     } = bnsName;
     await client.query(`UPDATE names SET latest = $1 WHERE name= $2`, [false, name]);
 
     await client.query(
       `
         INSERT INTO names(
-          name, address, registered_at, expire_block, zonefile_hash, zonefile, namespace_id, latest, tx_id, status, canonical, index_block_hash
-        ) values($1, $2, $3, $4, $5, $6, $7, $8,$9, $10, $11, $12)
+      name, address, registered_at, expire_block, zonefile_hash, zonefile, namespace_id, latest, tx_id, status, canonical, index_block_hash, atch_resolved
```

Comment:
I just noticed that in the bns related tables, `tx_id` is defined as a `string` rather than `bytea` column type. Can we update those?

Reply:
sure.